magma/docker/app/docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+export PATH="/app/node_modules/.bin:/app/vendor/bundle/$RUBY_VERSION/bin:$PATH"
+
+if [ -z "$SKIP_RUBY_SETUP" ]; then
+  bundle check || bundle install -j "$(nproc)"
+
+  rm -f tmp/pids/*.pid
+  if [ -z "$SKIP_DB_WAIT" ]; then
+    dockerize -wait tcp://magma_db:5432 -timeout 60s
+
+    for project in $(cat config.yml | grep ':project_path:' | sed -e 's/.*:project_path:\s*//g'); do
+      echo $project
+      bin/magma create_db "$(basename $project)"
+      MAGMA_ENV=test bin/magma create_db "$(basename $project)"
```

Comment:
I'm a little confused here. You set the `MAGMA_ENV` to `test` for these calls .... but shouldn't that just reflect the container `MAGMA_ENV` value? So when running the container for development, you may not have the `ipi` schema to run the migrations?

Reply:
I took out these four lines and the tests pass....but running the dev container results in DB errors. I think it's related to the shell grep / sed above finding dev and test projects.

```
Finished in 2.13 seconds (files took 0.96826 seconds to load)
258 examples, 0 failures

Randomized with seed 20558
```

Reply:
Yeah, I was lazy and added the invocations twice so that I could test both the test and development environments at once was all, so we can leave just one in.